sdk/typescript/src/cryptography/signature.ts

```diff
@@ -3,18 +3,22 @@
 
 import { fromB64, toB64 } from '@mysten/bcs';
 import type { PublicKey } from './publickey.js';
+import type { MultiSigStruct } from '../multisig/publickey.js';
+import { builder } from '../builder/bcs.js';
```

Comment:
Do you need the builder BCS (which has more stuff attached) or can you just use the plain sui BCS?

Reply:
We need to register the multisig types somehow, they are complex enough I wouldn't want to just duplicated it here.  Based on our conversation yesterday, I think it makes sense to just wait till bcs can be made more modular with refs